locations/spiders/omv.py

```diff
@@ -0,0 +1,187 @@
+import re
+
+import scrapy
+
+from locations.categories import Categories, Extras, Fuel, FuelCards, PaymentMethods, apply_category, apply_yes_no
+from locations.dict_parser import DictParser
+from locations.hours import DAYS, OpeningHours
+
+# List of brands and countries where they operate:
+# https://www.omv.com/en/customers/services/filling-stations
+BRANDS_AND_COUNTRIES = {
+    "OMV": {
+        "countries": ["AUT", "BGR", "HUN", "CZE", "ROU", "SRB", "SVK"],
+        "brand": "OMV",
+        "brand_wikidata": "Q168238",
+    },
+    "PETROM": {"countries": ["ROU", "MDA"], "brand": "Petrom", "brand_wikidata": "Q1755034"},
+    "AVANTI": {"countries": ["AUT"], "brand": "Avanti", "brand_wikidata": "Q168238"},
+    "HOFER": {"countries": ["AUT"], "brand": "Hofer Diskont", "brand_wikidata": "Q107803455"},
+}
+
+SITE_FEATURES_MAP = {
+    "ATM": Extras.ATM,
+    "Car wash hall": Extras.CAR_WASH,
+    "CNG": Fuel.CNG,
+    # TODO: map high flow pump, this seems an important attribute for petrol stations!
+    "High speed pump": None,
```

Comment:
May be related to HGVs. I know they have special pumps to allow fuel to flow faster. We need someone who drives and knows this stuff really.